R/preproc.R

```diff
@@ -1,19 +1,32 @@
+library(tidyverse)
+library(dplyr)
+library(fastDummies)
```

Comment:
don't include libraries in the function; the functions should be called with the "::" operator

Reply:
good catch, the is now updated.